habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -505,6 +505,7 @@ def step(self, human_joints_trans, **kwargs):
         """
         new_joints = human_joints_trans[:-16]
         new_pos_transform = human_joints_trans[-16:]
+        is_last_action = kwargs["is_last_action"]
```

Comment:
Make this passed directly to the function as a named argument rather than accessing it indirectly from `kwargs`.